lib/accelerometer.js

```diff
@@ -167,7 +167,7 @@ var Controllers = {
   MMA7361: {
     DEFAULTS: {
       value: {
-        zeroV: [336, 372, 287],
+        zeroV: [330, 350, 290],
```

Comment:
This could be avoided by defining the ESPLORA controller like this: 

```js
Controllers["ESPLORA"] = {
  DEFAULTS: {
    value: {
      zeroV: [x, y, z], 
      sensitivity: 170
    }
  }, 
  initialize: {
    value: function(opts, dataHandler) {
      opts.pins = [5, 11, 6];
      analogInitialize.call(this, opts, dataHandler);
    }
  },
  toGravity: Controllers.MMA7361.toGravity
};
```